src/AstroBase.jl

```diff
@@ -485,4 +489,60 @@ function xy06(jd1, jd2)
     sec2rad((xpr + (xyls[1] + xypl[1]) / 1e6)), sec2rad(ypr + (xyls[2] + xypl[2]) / 1e6)
 end
 
+"""
+    nutation(jd1, jd2)
+
+Returns nutation in longitude(radians) and obliquity(radians) for a given 2 part Julian date (TT format).
+
+# Example
+
+```jldoctest
+julia> nutation(2.4578265e6, 0.30434616919175345)
+(-3.7565297299394694e-5, -3.665617105048724e-5
+```
+"""
+function nutation(jd1, jd2)
+
+    t = ((jd1 - J2000) + jd2) / DAYS_PER_CENTURY
+
+    mean_longitude_moon_minus_mean_longitude_moon_perigee = rem2pi(
```

Comment:
So you decided to discard `normalize_pi_angle` function?  It's fine with me as its definition is quite simple, but I think it wouldn't hurt either.

Reply:
Yes, I thought it was causing verbosity. Should I reimplement it?

Reply:
It's fine with me to keep this way, but let @helgee decide :sweat_smile:

Reply:
Sure!

Reply:
Let's keep it this way. 👍